schema/spacewalk/upgrade/susemanager-schema-4.4.6-to-susemanager-schema-4.4.7/002-store-vulnerable-packages.sql

```diff
@@ -0,0 +1,68 @@
+CREATE TABLE IF NOT EXISTS suseOVALVulnerablePackage
+(
+    id             NUMERIC NOT NULL
+        CONSTRAINT suse_oval_vulnerable_pkg_id_pk PRIMARY KEY,
+    name           VARCHAR NOT NULL,
+    fix_version    VARCHAR
+);
+
+CREATE SEQUENCE IF NOT EXISTS suse_oval_vulnerable_pkg_id_seq START WITH 301;
+
+CREATE UNIQUE INDEX IF NOT EXISTS suse_oval_vulnerable_pkg_name_fix_version ON suseOVALVulnerablePackage(name, fix_version);
+
+CREATE TABLE IF NOT EXISTS suseOVALPlatformVulnerablePackage
```

Comment:
Some about this table, looks duplicated.

Reply:
This too.